index.d.ts

```diff
@@ -0,0 +1,52 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+declare module "mocha-testdata" {
```

Comment:
Hmmm, I don't think so...  If you look at the [ambient Knockout deifnition file](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7de6c3dd94feaeb21f20054b9f30d5dabc5efabd/knockout/knockout.d.ts#L103-L111), you can see that it isn't `export`ed, but it was usable in our code.